resources/views/events/index.blade.php

```diff
@@ -71,48 +50,49 @@
         @endif
         {{-- END Events to Moderate (Admin Only) --}}
 
-        {{-- Upcoming events for your Groups --}}
```

Comment:
Likewise we have lost Upcoming events from /party ?

Reply:
I missed this function to show all events from your groups.  Have added.